.codecov.yml

```diff
@@ -1,3 +1,7 @@
 codecov:
   branch: next
 comment: false
+
+coverage:
+  ignore:
+    - ".*"
```

Comment:
Is that change breaking the code coverage?
https://codecov.io/github/callemall/material-ui/commit/db7e18a0ede1a8a3c0e2441787fa1311c552b3e0

Reply:
Yes. I was trying to ignore dotfiles, I cannot merge this config change because it falsely reduces code coverage. I'll figure out what is wrong with the pattern.

Reply:
I have reverted the ignore, if codecov is low, we need to force merge.